invokeai/app/api_app.py

```diff
@@ -120,7 +122,7 @@ def custom_openapi():
 app.openapi = custom_openapi
 
 # Override API doc favicons
-app.mount("/static", StaticFiles(directory="static/dream_web"), name="static")
+app.mount("/static", StaticFiles(directory=Path(web_dir.__path__[0], 'static/dream_web')), name="static")
```

Comment:
Where does `web_dir` come from? It doesn't exist as far as I can tell

Reply:
So sorry. This should never have worked and I don't understand how it seemed to work when I tested it. I have added the missing line:

```import invokeai.frontend.web as web_dir```